token/cli/src/main.rs

```diff
@@ -3636,6 +3647,48 @@ mod tests {
         assert_eq!(ui_account.token_amount.amount, "90");
     }
 
+    #[test]
+    fn close_wrapped_sol_account() {
+        let (test_validator, payer) = validator_for_test();
+        let config = test_config(&test_validator, &payer, &spl_token::id());
+        let bulk_signers: Vec<Box<dyn Signer>> = vec![Box::new(clone_keypair(&payer))];
+
+        let token = create_token(&config, &payer);
+        let source = create_associated_account(&config, &payer, token);
+        let ui_amount = 10.0;
+        command_wrap(
+            &config,
+            ui_amount,
+            payer.pubkey(),
+            Some(source),
+            bulk_signers,
+        )
+        .unwrap();
```

Comment:
This won't do exactly what you expect -- you're creating a new mint and account, and then trying to wrap that account that was just created.  To do this simply. you can just wrap the default account by doing
```suggestion
        let token = create_token(&config, &payer);
        let source = create_associated_account(&config, &payer, token);
        let ui_amount = 10.0;
        command_wrap(
            &config,
            ui_amount,
            payer.pubkey(),
            None,
            bulk_signers,
        )
        .unwrap();
```